### PR TITLE
add the organisational header variant

### LINKED ISSUE #161: Header: add the organisational header variants
# Organisational header

https://service-manual.nhs.uk/design-system/components/header#organisational-header

- [x] Ability to choose between the standard NHS header or the organisational header
- [x] Ability to change the organisation name and organisation descriptor in admin
- [x] Ability to upload a custom .png/jpg image in admin
- [x] Ability to choose between the blue or white header in admin

Potentially related issue: https://github.com/nhsuk/wagtail-nhsuk-frontend/issues/132

I think this functionality will open up this plugin to more users

### PR DESCRIPTION
**Pull request by chrimesdev**
- Ability to use the organisation header with organisation name and descriptor and the white variant.
- Ability to upload a custom logo to replace the default NHS logo, which can be used for organisation logos.
- Reorganised some of the admin layout to be clearer and organised.

To do:

- [x] Add the organisation split element, to split larger organisation names
- [x] Review admin layout, settings and labels 
- [x] Test different settings variations
- [x] Test the logo sizing for custom logo

Notes

There was an issue when enabling both Service header and Transactional header that both of the HTML would be generated. The general rule of thumb is that any settings lower down the page will overwrite those further up the page (similar to how CSS works).

fixes #161

### CODE REVIEW

**Comment by DomBaker** [comment]
Fixed it for you @chrimesdev 😉

**Review by mikemonteith — CHANGES_REQUESTED** [review]
Left comments inline.
Also need to update the documentation at [./docs/components/header.md](https://github.com/nhsuk/wagtail-nhsuk-frontend/blob/master/docs/components/header.md)

File: wagtailnhsukfrontend/settings/migrations/0009_merge_0006and0008.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.8 on 2021-05-10 13:09
```
[the remark this answers is not in the document]
Comment (DomBaker) [reply wagtailnhsukfrontend/settings/migrations/0009_merge_0006and0008.py]: Fixed this issue.

File: wagtailnhsukfrontend/settings/models.py [hunk 2]
```diff
@@ -38,21 +40,40 @@ class HeaderSettings(ClusterableModel, BaseSetting):
         blank=True,
         help_text="Aria label override for the NHS logo."
     )
+    logo_custom = models.ForeignKey(
+        Image,
```
[the remark this answers is not in the document]
Comment (mikemonteith) [reply wagtailnhsukfrontend/settings/models.py:44]: According to the docs, wagtail plugins should always use `get_image_model()`

File: wagtailnhsukfrontend/settings/models.py [hunk 3]
```diff
@@ -38,21 +40,40 @@ class HeaderSettings(ClusterableModel, BaseSetting):
...
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
 
     show_search = models.BooleanField(default=False)
 
+    organisation_name = models.CharField(max_length=255, blank=True)
```
[the remark this answers is not in the document]
Comment (DomBaker) [reply wagtailnhsukfrontend/settings/models.py:53]: Added some help text for both of the queried fields, took the information from the service manual.

File: wagtailnhsukfrontend/settings/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (DomBaker) [reply wagtailnhsukfrontend/settings/models.py]: updated.

**Review by mikemonteith — APPROVED** [review]

**Comment by damonjagger** [comment]
🔥

**Pull request merged**